xml/en/security_advisories.xml

```diff
@@ -14,7 +14,9 @@
 
 <para>
 All nginx security issues should be reported to
-<link url="mailto:security-alert@nginx.org">security-alert@nginx.org</link>.
+<link url="mailto:F5SIRT@f5.com">F5SIRT@f5.com</link>
```

Comment:
Placing here the email address duplicates content of SECURITY.md, just leave SECURITY.md link here.

Reply:
This was made on purpose - if the security.md page couldn't be opened there's still a point of contact available.